Projektarbeit III/Assets/Scripts/Player/PlayerController/JumpingState.cs

```diff
@@ -27,8 +33,16 @@ public void OnEnter()
 
     public void UpdateState()
     {
-        // Transition to IdleState when the player starts falling
+        MovementUtils.ApplyHorizontalMovement(rb, movementAction, moveSpeed);
+
+        // Apply gravitational pull after reaching the peak of the jump
         if (rb.linearVelocity.y <= 0)
+        {
+            rb.linearVelocity += Vector2.down * fallForce * Time.deltaTime;
```

Comment:
This seems a little overly complex..
```suggestion
            rb.linearVelocityY -= fallForce * Time.deltaTime;
```
should work just fine